lib/elixir/test/elixir/kernel/warning_test.exs

```diff
@@ -981,7 +981,7 @@ defmodule Kernel.WarningTest do
 
   # TODO: Simplify when we require Erlang/OTP 24
   if System.otp_release() >= "24" do
-    @argument_error_message "the call to :erlang.atom_to_binary/2"
+    @argument_error_message "the call to :erlang.atom_to_binary/1"
```

Comment:
When you inline, we can now do a reverse mapping, so showing Atom.to_string/1 is the correct now! And that's good because it is most likely what the user wrote!

Reply:
That's great! I was wondering if there was some mechanism like that in the back :)